src/components/chips/chips.spec.js

```diff
@@ -659,6 +659,59 @@ describe('<md-chips>', function() {
 
       });
 
+      describe('required', function() {
+        beforeEach(function() {
+          // Clear default items to test the required chips functionality
+          scope.items = [];
+        });
+
+        it('should set the required error when chips is compiled with an empty array', function() {
+          var template =
+              '<form name="form">' +
+              '<md-chips name="chips" required ng-model="items"></md-chips>' +
+              '</form>';
+
+          var element = buildChips(template);
+          element.scope().$apply();
+
+          expect(scope.form.chips.$error['required']).toBe(true);
```

Comment:
You can use `scope.form.chips.$error.required`, instead of the bracket notation here and the other tests.